code/modules/mod/mod_theme.dm

```diff
@@ -1057,15 +1037,15 @@
 	//energy = 15 // 25
 
 /datum/mod_theme/prototype
-	name = "prototype"
-	desc = "A prototype modular suit powered by locomotives. While it is comfortable and has a big capacity, it remains very bulky and power-inefficient."
-	extended_desc = "This is a prototype powered exoskeleton, a design not seen in hundreds of years, the first \
-		post-void war era modular suit to ever be safely utilized by an operator. This ancient clunker is still functional, \
-		though it's missing several modern-day luxuries from updated Cybersun Industries designs. \
-		Primarily, the suit's myoelectric suit layer is entirely non-existant, and the servos do very little to \
-		help distribute the weight evenly across the wearer's body, making it slow and bulky to move in. \
-		The internal heads-up display is rendered in nearly unreadable cyan, as the visor suggests, \
-		leaving the user unable to see long distances. However, the way the helmet retracts is pretty cool."
+	name = "'Ogre' prototype"
+	desc = "A prototype modular suit developed many years ago. Incredibly heavy, power inefficient, and lacking in protection. At the very least, it's easy to modify."
+	extended_desc = "Quite possibly the first true modsuit ever created, the 'Ogre', as its mostly-faded ID label calls it, is a positively ancient mechanized suit that represents the very beginning of \
+		the Sector's foray into powered modsuits. Naturally, such an elderly design lacks virtually all of the quality of life features seen in modsuits today. \
+		The first-generation servodrives are woefully underpowered and exceptionally inefficient, leading to a ponderous top speed and an average battery life entirely \
+		unsuited to extended EVA usage. As little more than technology testbed, the Ogre is also sorely lacking in armor protection, possessing little more than a steel carapace \
+		more suited to deflecting micrometeoroids than any dedicated weaponry, though the treated underlayer does offer effective fire and chemical protection, at the cost of greatly \
+		increased cancer risks after several hours of sustained use. Today, it is exceptionally rare to find this suit anywhere outside a museum exhibit or disused \
+		warehouses, but isolated models can still sometimes be found in abandoned storage units floating through the void, waiting for a desperate salvager to claim them."
```

Comment:
You might want to remove references to multiples of this existing as it is a theta specific prototype.

You gonna have to go and make a special PR just for this where you also edit theta because of the RND notes on this modsuit that spawn in the vault next to it.